### ISSUE TITLE
this is so detailed, what websites/apps would you use instead of discord btw?

### ISSUE THREAD

**Issue by Scripped** [Its-Vichy/lets-talk-about-discord#8]

**Comment by Scripped** [comment]
also could you add that discord support doesn't help anyone

**Comment by hoemotion** [comment]
If you wanna self-host discord I would check out [fosscord](https://fosscord.com/), there are also some similar apps like [guilded](https://guilded.gg), [nertivia](https://nertivia.net/) and [revolt](https://revolt.chat) but revolt is pretty buggy

**Comment by hoemotion** [comment]
didnt mention [kahelia](https://www.kaiheila.cn/) since we need a chinese number for using this only-chinese app which looks exactly like discord 💀

**Comment by Scripped** [comment]
tysm!! 😊

**Comment by jdghreiongbergm** [comment]
> tysm!! 😊

use something like matrix or telegram. or even signal.

**Comment by Cipher-GF** [comment]
matrix is nice too

**Comment by I2rys** [comment]
> matrix is nice too

Yeah Matrix is really good It's just not good for communities though unlike Discord, Fosscord, Revolt & etc also can I be in the "Trusted developers"? ^^
